src/releasemonitor.cpp

```diff
@@ -39,7 +39,15 @@ void ReleaseMonitor::runInternal() {
           &ReleaseMonitor::updateRequired);
   connect(updater, &Updater::updateRecommended, this,
           &ReleaseMonitor::updateRecommended);
-  connect(updater, &QObject::destroyed, this, &ReleaseMonitor::releaseChecked);
+  connect(updater, &Updater::updateRequired, this,
+          &ReleaseMonitor::updateRequiredOrRecommended);
+  connect(updater, &Updater::updateRecommended, this,
+          &ReleaseMonitor::updateRequiredOrRecommended);
+  connect(updater, &Updater::updateNotAvailable, this,
+          &ReleaseMonitor::updateNotAvailable);
+  connect(updater, &QObject::destroyed, [this]() {
```

Comment:
This can be as it was before: `connect(updater, &QObject::destroyed, this, &ReleaseMonitor::releaseChecked);` or it does not work?